packages/integrations/netlify/src/index.ts

```diff
@@ -662,6 +685,15 @@ export default function netlifyIntegration(
 						sharpImageService: 'stable',
 						envGetSecret: 'stable',
 					},
+					client: {
+						internalFetchHeaders: (): Record<string, string> => {
+							const deployId = process.env.DEPLOY_ID;
+							if (deployId) {
+								return { 'X-Netlify-Deploy-ID': deployId };
+							}
+							return {};
```

Comment:
Netlify should get `assetQueryParams` too (which would need adding to `skew-protection.json`)